build.gradle.kts

```diff
@@ -539,7 +539,8 @@ tasks.rat {
     "**/LICENSE.*",
     "**/NOTICE.*",
     "ROADMAP.md",
-    "clients/client-python/.pytest_cache/*"
+    "clients/client-python/.pytest_cache/*",
+    "clients/client-python/gravitino.egg-info/*"
```

Comment:
This folder does not exist on my local machine, can anyone help to confirm this point?

Reply:
Try gradle build.